modules/op-release-notes-1-3.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assembly:
+//
+// * openshift_pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-3_{context}"]
+= Release notes for {pipelines-title} Technology Preview 1.3
+
+[id="new-features-1-3_{context}"]
+== New features
+{pipelines-title} Technology Preview (TP) 1.3 is now available on {product-title} 4.7. {pipelines-title} TP 1.3 is updated to support:
+
+* Tekton Pipelines 0.19.0
+* Tekton `tkn` CLI 0.15.0
+* Tekton Triggers 0.10.2
+* ClusterTasks based on Tekton Catalog 0.19.0
+* IBM Power Systems on {product-title} 4.7
+* IBM Z and LinuxONE on {product-title} 4.7
+
+In addition to the fixes and stability improvements, here is a highlight of what’s new in OpenShift Pipelines 1.3.
```

Comment:
improvements, the following sections highlight what is new in in OpenShift Pipelines 1.3.